canonicalwebteam/blog/common_view_logic.py

```diff
@@ -241,12 +243,33 @@ def get_archives(self, page=1, group="", month="", year="", category=""):
 
         return context
 
-    def get_feed(self, uri):
-        feed = api.get_feed(self.tag_name)
-        right_urls = logic.change_url(feed, uri.replace("/feed", ""))
-        context = right_urls.replace("Ubuntu Blog", self.blog_title)
+    def get_feed(
+        self, uri, tags_exclude=[], tags=[], title="Ubuntu Blog", subtitle=""
```

Comment:
Can we default title to something more generic or make it required? "Ubuntu Blog" is quite specific.

Reply:
Changed it to "Blog"